src/main/java/io/cdap/delta/bigquery/BigQueryTarget.java

```diff
@@ -186,6 +186,14 @@ private static String stringifyPipelineId(DeltaPipelineId pipelineId) {
     @Description("Whether to require manual intervention when a drop table or drop database event is encountered.")
     private Boolean requireManualDrops;
 
+    @Nullable
+    @Description("Optional. By default the dataset name is same as source database name.")
```

Comment:
Would be good to document the naming convention that need to be followed. For example : `valid name should only contain letters , numbers, and underscores and maximum length can be 1024 chars. Any invalid chars would be replaced by _ in the final dataset name`

Reply:
good point :)